src/UI/Seller/src/app/shared/components/sort-table-header/sort-table-header.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { faCaretUp, faCaretDown } from '@fortawesome/free-solid-svg-icons';
+
+@Component({
+  selector: 'shared-sort-table',
+  templateUrl: './sort-table-header.component.html',
+  styleUrls: ['./sort-table-header.component.scss'],
+})
+export class SortTableHeaderComponent {
+  constructor() {}
+  faCaretUp = faCaretUp;
+  faCaretDown = faCaretDown;
+
+  // The text to display to the user on the header for this column
+  @Input() displayName: string;
+
```

Comment:
A better name might be `shared-sort-column`, i was a little confused when i ran into this in the html first and looked like an array of tables inside a table tag ;)

Reply:
and actually maybe just call it column? Looks like sort is optional.

Reply:
My intention is that if you use this component you always get the ability to sort by that column.

Reply:
I like 'shared-sort-column'